### ISSUE TITLE
nvcc_wrapper --host-only fails with mutiple -W* flags

### ISSUE THREAD

**Issue by lkersting** [kokkos/kokkos#2484]
When trying to compile using the nvcc_wrapper and multiple -W* flags (e.g. -Wall, -Wextra, etc) I'm getting a compiler error. The `xcompiler_args` passed to the `host_command` are comma separated (which they should be for nvcc_command), but the host compiler requires them to be space separated. A simple solution would be to add:
~~~~~~~~~~~
# Modify xcompiler_args for host only
if [ $host_only -eq 1 ]; then
  # Replace all commas with a space
  xcompiler_args=${xcompiler_args//,/" "}
fi
~~~~~~~~~~~
after parsing all the arguments.

**Comment by crtrott** [comment]
which host compiler

**Comment by lkersting** [comment]
I'm using GNU 7.3.0

**Comment by masterleinad** [comment]
I can confirm this:
```
$ ./bin/nvcc_wrapper --host-only -Werror -Wall /tmp/cuda-dummy.cu 
g++: error: unrecognized command line option ‘-Werror,-Wall’; did you mean ‘-Wno-all’?
```

**Comment by crtrott** [comment]
@masterleinad can you fix that?

**Comment by masterleinad** [comment]
Addressed in #2487.
